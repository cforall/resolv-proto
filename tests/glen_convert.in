// Based on example in pp.92-93 of Glen's thesis
// 
// Expected result is f( not-i( 1 ) => 3 ) (e.g. break total-cost ties based on local cost) 
1 not-i 1
2 not-ui 2
3 not-ul 3
f 3
%%
f( not( 1 ) )
f( not( not( 1 ) ) )
