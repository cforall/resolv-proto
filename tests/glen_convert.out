
[Void / (0,0,0,0,2)] f( not-i( 1 ) => 3 )
